.all-contributorsrc

```diff
@@ -117,6 +121,13 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/61125174?v=4",
       "profile": "https://github.com/ho-ssain",
       "contributions": ["doc"]
+    },
+    {
+      "login": "kabrunko-dev",
+      "name": "Diogo Nishikawa",
+      "avatar_url": "https://avatars.githubusercontent.com/u/142346548?v=4",
+      "profile": "https://github.com/kabrunko-dev/",
+      "contributions": ["translation-pt-BR"]
```

Comment:
You don't have to write this. 😅 It's done on a follow up PR and it's done automatically with all-contributors CLI.

Reply:
Hahaha sorry. I'm still new in the OSS world. But I'll keep it in mind now

Reply:
no problem 😂